### DISCUSSION TITLE
Random Quoter Design 1.0

### DISCUSSION THREAD

**Discussion by MaximusI30** [AndreyDmitriev39r/random-quote-machine-v2#12] [Ideas]
Here is my design variation of our [project](https://www.figma.com/file/9CrFqV2tdbPiYoPm3kSCGZ/random-quote-machine-v2?type=design&node-id=0%3A1&mode=design&t=y2M0IALpgj8EeK9L-1) in figma. 
Guys, please take a look and share your thoughts.

**Comment by AndreyDmitriev39r** [comment]
Good to go, we can use it for initial design.
I am gonna create some issues based on that.
